test/unittest/statistics/dds/StatisticsQosTests.cpp

```diff
@@ -63,20 +63,128 @@ class StatisticsFromXMLProfileTests : public ::testing::Test
+                                            <announcement_period>                                                           \
+                                                    <sec>1</sec>                                                            \
+                                                    <nanosec>856000</nanosec>                                               \
+                                            </announcement_period>                                                          \
+                                    </liveliness>                                                                           \
+                            </qos>                                                                                          \
+                    </data_writer>                                                                                          \
+                    <data_writer profile_name=\"OTHER_NAME_FOR_PROFILE\">                                                   \
+                            <qos>                                                                                           \
+                                    <reliability>                                                                           \
+                                            <kind>BEST_EFFORT</kind>                                                        \
+                                    </reliability>                                                                          \
+                            </qos>                                                                                          \
+                    </data_writer>                                                                                          \
+                    <data_writer profile_name=\"DISCOVERY_TOPIC\">                                                          \
+                            <qos>                                                                                           \
+                                    <reliability>                                                                           \
+                                            <kind>RELIABLE</kind>                                                           \
+                                    </reliability>                                                                          \
+                            </qos>                                                                                          \
+                    </data_writer>                                                                                          \
+                </profiles>                                                                                                 \
+            </dds>                                                                                                          \
+        ";
```

Comment:
Is the alignment forced by uncrustify? I do not think so. I suggest modifying and using our tabulations rules (only 4 spaces between levels instead of the 8 being applied now.

Reply:
The alignment was the one the file had before. However, I will apply a 4 indent for uniformity.

Reply:
Thanks! I know it is legacy code, but in this way we strive to always be improving the code. :smile: